TYPO3-Solr/ext-solr
# [FEATURE] Add and improve translations

# What this pr does

To simplify the usage of EXT:solr in multi language installations,
translations will be extended and improved.

This commit provides new translations for Spanish and Portuguese,
existing translations for French, Japanese, Korean and Chinese are
extended/fixed.

# How to test

Configure on of the new or adapted languages in backend and frontend and check the displayed translations.

Resolves: #2874
